week1/src/backend/routes/reservation.js

```diff
@@ -0,0 +1,7 @@
+const reservations = require("../data/reservations.json");
+
+let reservation = function(){
+    return  reservations[Math.floor(Math.random() * reservations.length)];
```

Comment:
You are doing a lot here! Try divide it into steps: 
```suggestion
    const randomIndex = Math.floor(Math.random() * reservations.length);
    return  reservations[randomIndex ];
```
It makes it easier to read as a developer